cli/src/pcluster/resources/imagebuilder/parallelcluster.yaml

```diff
@@ -197,7 +197,21 @@ phases:
           commands:
             - |
               set -v
+              PLATFORM='{{ build.PlatformName.outputs.stdout }}'
+
+              if [[ ${!PLATFORM} == RHEL ]]; then
+                CA_CERTS_FILE=/etc/ssl/certs/ca-bundle.crt
+              elif [[ ${!PLATFORM} == DEBIAN ]]; then
+                CA_CERTS_FILE=/etc/ssl/certs/ca-certificates.crt
+              fi
+
               curl --retry 3 -L {{ build.CincUrl.outputs.stdout }} | bash -s -- -v {{ ChefVersion }}
+
+              if [[ -e ${!CA_CERTS_FILE} ]]; then
+                rm -f /opt/cinc/embedded/ssl/certs/cacert.pem
+                ln -s ${!CA_CERTS_FILE} /opt/cinc/embedded/ssl/certs/cacert.pem
```

Comment:
minor - What about using `ln -sf` and removing the previous `rm`?

Reply:
Sounds good, I'll push it.

Reply:
It's done. I also added an entry in the ChangeLog.